src/stcal/ramp_fitting/utils.py

```diff
@@ -1464,6 +1478,12 @@ def compute_median_rates(ramp_data):
     group_time = ramp_data.group_time
     frame_time = ramp_data.frame_time
     adjustment = group_time / frame_time
+
+    if ramp_data._1ggroups_time is not None:
+        _1ggroups_time_adjustment = group_time / ramp_data._1ggroups_time
```

Comment:
Minor nit, but any particular reason why the name `_1ggroups_time_adjustment` should have the style of a private variable (i.e. the leading underscore)? We don't have that for the equivalent adjustment for zeroframe, which is just `adjustment`. By the way, because we have 2 different "adjustments" now, I think it would be good to give each of them a name related to their specific function, i.e. change `adjustment` to `zeroframe_time_adjustment` (to have similar syntax as the group 1 adjustment).

Reply:
I mixed up my coding styles and chose a poor variable name.  I've changed it to remove this to better conform to python style.